roles/edpm_ovs/tasks/update.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Update openvswitch packages without stopping the service
+  become: true
+  tags:
+    - update
+    - ovs
+  ansible.builtin.shell: >
+    dnf update -y openvswitch --setopt=tsflags=noscripts
+  register: update_ovs_result
+  failed_when: update_ovs_result.rc != 0
+
+- name: Reload openvswitch service
```

Comment:
I think in general, we should NOT reload the service. We may have an additional flag to reload it too (needed e.g. when there's a security patch to apply; disabled by default).

Reply:
I thought we discussed that we should reload after every update just in case. This reload will use the `flow-restore-wait` option and thus will not cause any recompute. But we can go without doing a reload. At this point I dont see one option being better over the other.

Reply:
I believe the wording in the design document was that unless we can't skip restarting the process, then we should "reload". But ideally, we don't reload without a reason.